sormas-backend/src/main/java/de/symeda/sormas/backend/report/AggregateReportService.java

```diff
@@ -93,9 +96,7 @@ public Predicate createUserFilter(CriteriaBuilder cb, CriteriaQuery cq, From<Agg
 		Predicate filter = cb.equal(reportingUser, currentUser);
 
 		// Allow access based on user role
-		if (currentUser
-			.hasAnyUserRole(UserRole.SURVEILLANCE_SUPERVISOR, UserRole.CONTACT_SUPERVISOR, UserRole.CASE_SUPERVISOR, UserRole.STATE_OBSERVER)
-			&& currentUser.getRegion() != null) {
+		if (jurisdictionLevel == JurisdictionLevel.REGION && currentUser.getRegion() != null) {
```

Comment:
Probably need to check for isPortHealthUser here as well (to be consistent with the check above)

Reply:
ok, but that means i'm changing the existing functionality

Reply:
Didn't you already change it a few lines above though?

Reply:
no, i changed the if's using the new enum, but the actual checking done by the if should remain the same, at least that is the intention